src/Bicep.Local.Deploy/Extensibility/LocalExtensibilityExtension.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Deployments.Engine.Host.Azure.ExtensibilityV2.Contract.Models;
+using Azure.Deployments.Extensibility.Contract;
+using Azure.Deployments.Extensibility.Messages;
+
+namespace Bicep.Local.Deploy.Extensibility;
+
+public abstract class LocalExtensibilityExtension : IAsyncDisposable
```

Comment:
`...ExtensibilityExtension` is a bit strange - what about `LocalDeployExtension`?

Reply:
Agree that the class functions more as an extension host / dispatcher. I think either 'LocalExtensibilityHost' or 'LocalExtensionHost' would be suitable names.